projects/plugins/jetpack/modules/likes.php

```diff
@@ -521,7 +521,7 @@ public function admin_bar_likes() {
 		// Make sure to include the scripts before the iframe otherwise weird things happen.
 		add_action( 'wp_footer', 'jetpack_likes_master_iframe', 21 );
 
-		$src = sprintf( 'https://widgets.wp.com/likes/#blog_id=%2$d&amp;post_id=%3$d&amp;origin=%1$s://%4$s', $protocol, $blog_id, $post_id, $domain );
+		$src = sprintf( 'https://widgets.wp.com/likes/?ver=%1$d#blog_id=%3$d&amp;post_id=%4$d&amp;origin=%2$s://%5$s', JETPACK_VERSION, $protocol, $blog_id, $post_id, $domain );
```

Comment:
This change needs to be checked - as I believe we haven't yet tested our previous changes on the Like widget as as part of Admin bar,